backend/infrahub/core/schema/schema_branch.py

```diff
@@ -1468,6 +1494,30 @@ def generate_weight(self) -> None:
 
             self.set(name=name, schema=node)
 
+        for name in self.node_names + self.profile_names:
+            node = self.get(name=name, duplicate=False)
+
+            items_to_update = [item for item in node.attributes + node.relationships if not item.order_weight]
+
+            if not items_to_update:
+                continue
+            node = node.duplicate()
+
+            generic_fields_map = self._get_generic_fields_map(node_schema=node)
+
+            current_weight = 0
```

Comment:
I know it might be a bit unrelated to this PR but I'm wondering if we should change the starting value of 0 here, it feels like the starting weight should be the highest weight that's defined within the schema of the current node. It would seem less random than 0.